src/cpp/shared_core/include/shared_core/FilePath.hpp

```diff
@@ -573,6 +573,14 @@ class FilePath
     */
    Error openForWrite(std::shared_ptr<std::ostream>& out_stream, bool in_truncate = true) const;
 
+   /*
+    * @brief Checks if a file can be written to by opening the file
```

Comment:
Nit: periods at the ends of the sentences and a blank comment line between the @brief and long descriptions.

Reply:
Fixed 👍